docs/routing/introduction.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+Most JavaScript frameworks come with a dedicated routing solution
+like `angular/router` or `vue-router`. They make it possible to
+navigate through different pages of an application without having to
+do a full page refresh on every link click.
+Since we no longer have one monolithic application that handles all routes, instead we have several independent applications, we need to solve an important issue, namely routing.
+
+The following describes how this is handled in the ILC, but first let's clear up the terminology:
+
+ - Hard transitions - A transition handled by ILC, with such a transition, the application on the page changes to another one.
+
+ - Soft transitions - A transition that is handled inside the application, with such a transition, only the content inside the application changes, but not the application itself.
+
+  ![Introdaction demo](../assets/routes/introduction-demo.png)
+
+#
+
+Now, let's talk a little about the theory, after which we will move on to practical examples:
+
+ In ILC, we can use one HTML template for all of our applications. With this approach, SSR occurs once when the page is first loaded, then all navigation occurs through CSR. So let's look at navigation in more detail:
```

Comment:
"we can use one HTML template" - no, we use always one template, and we don't have the capability to change it. Please correct this sentence.